src/classes/class-typography.php

```diff
@@ -0,0 +1,381 @@
+                                $typography_css['editor'] .= $typography_styles;
+                            } else {
+                                $typography_css['front'] .= $typography_styles;
+                            }
+                        }
+                    }
+                }
+            }
+        }
+
+        return $typography_css;
+    }
+
+    /**
+     * Check value on the existence and emptiness.
+     *
+     * @param void   $value - Checking value.
+     * @param bool   $attribute - Checking attribute of Array Value.
+     * @param string $mode - Full or isset for partial check.
+     * @return bool  $value - True or false.
+     */
+    public function is_exist( $value, $attribute = false, $mode = 'full' ) {
+        $check = false;
+        if ( $attribute ) {
+            if ( 'full' === $mode && isset( $value[ $attribute ] ) && ! empty( $value[ $attribute ] ) ) {
+                $check = true;
+            }
+            if ( 'isset' === $mode && isset( $value[ $attribute ] ) ) {
+                $check = true;
+            }
+        } else {
+            if ( 'full' === $mode ) {
+                $check = ( isset( $value ) && ! empty( $value ) ) ? true : false;
+            }
+            if ( 'isset' === $mode ) {
+                $check = ( isset( $value ) ) ? true : false;
+            }
+        }
+        return $check;
+    }
+
+    /**
+     * Function for get Typography Values.
+     *
+     * @param object $typography_object - Current typography.
+     * @param array  $typography_prepeare_styles - Previous Array With Current Styles Properties.
+     * @return mixed - Next Array With Current Styles Properties.
+     */
+    public function get_typography_values( $typography_object, $typography_prepeare_styles ) {
+        $conformity_attributes = array(
+            'fontFamily' => 'font-family',
+            'fontFamilyCategory' => 'font-family-category',
+            'fontSize' => 'font-size',
+            'fontWeight' => 'font-weight',
+            'lineHeight' => 'line-height',
+            'letterSpacing' => 'letter-spacing',
+            'label' => 'label',
+            'childOf' => 'child_of',
+        );
+        if ( $this->is_exist( $typography_object ) ) {
+            foreach ( json_decode( $typography_object ) as $meta_typography_key => $meta_typography_value ) {
+                if ( $this->is_exist( $typography_prepeare_styles[ $meta_typography_key ], false, 'isset' ) ) {
+                    foreach ( $meta_typography_value as $typography_attribute_key => $typography_attribute ) {
+                        if ( $this->is_exist( $conformity_attributes[ $typography_attribute_key ] ) &&
+                            $this->is_exist( $typography_prepeare_styles[ $meta_typography_key ]['style-properties'], $conformity_attributes[ $typography_attribute_key ], 'isset' ) ) {
+                            if ( 'Default Site Font' !== $typography_attribute && '' !== $typography_attribute ) {
```

Comment:
We can't use `Default Site Font` in conditions and save this value in DB because it is translatable Label for Select component. The value should be an empty string.